modules/zivid/experimental/_pixel_mapping.py

```diff
@@ -0,0 +1,50 @@
+"""Module for experimental pixel mapping. This API may change in the future."""
+
+import _zivid
+
+
+class PixelMapping:
+    """Pixel mapping from subsampled to full resolution.
+
+    Required when mapping an index in a subsampled point cloud to e.g. a full resolution 2D image.
+    """
+
+    def __init__(self, row_stride=1, col_stride=1, row_offset=0, col_offset=0) -> None:
+        self.__impl = _zivid.PixelMapping(
+            row_stride, col_stride, row_offset, col_offset
+        )
+
+    @property
+    def row_stride(self):
+        return self.__impl.row_stride()
+
+    @row_stride.setter
+    def row_stride(self, value):
+        self.__impl.row_stride = value
```

Comment:
Do we need these setters? We dont have that in C++? Just the consturctor.

Reply:
I agree, leave it to only getters for now.

Reply:
https://github.com/zivid/zivid-python/pull/251/commits/65668230e040186f33ed889953a8bc14c70ac58e